HexCalc.java

```diff
@@ -0,0 +1,272 @@
+/*
+Programme to perform Calculations Between Hexadecimal Numbers 
+Addition Substractions Multiplications Dividation
+
+*/
+import java.util.*;
+class HexCalc
+{	
+//method to convert char to corresponding intiger 
+    public static int val(char c) 
+    { 
+        if (c >= '0' && c <= '9') 
+            return (int)c - '0'; 
+        else
+            return (int)c - 'A' + 10; 
+    } 
+ //valid string check
+    public static boolean checkValid(String s,int base)
+    {
+    	for (int i = 0; i<s.length()-1; i++) 
+        {
+            if(val(s.charAt(i))>=base)
```

Comment:
Follow proper indentation. Do it for all other occurrences.

Reply:
ok sir